polar_route/route_planner.py

```diff
@@ -15,7 +15,9 @@
 from pandas.core.common import SettingWithCopyWarning
 warnings.simplefilter(action="ignore", category=SettingWithCopyWarning)
 
-from polar_route.crossing import NewtonianDistance, NewtonianCurve
+from polar_route.crossing import NewtonianDistance
+
+from polar_route.crossing_smoothing import Smoothing,PathValues,find_edge
 
 
```

Comment:
is there any reason why these functions aren't inside the `RoutePlanner` object? If not, maybe they could be moved to the utils.py file?

Reply:
This is beyond the #218 and should be corrected in an additional issue as it requires testing of cli and additional submodules that will change

Reply:
I don't personally think there's any reason this should block the merge into `0.2.x` since we plan to refactor the route planner code in the next few releases anyway